guardian/newsletters-nx
# Combine tag and sign-up creation emails to cp

## What does this change?

Combines the existing Tag creation and sign-up page email requests in to one. Adds a news email config for the CP recipients

## How to test

Update dev / code config to ensure you emails for the `centralProductionRecipients` group. Create and launch a newsletter and ensure that.

## How can we measure success?

Email received as expected on launch

## Have we considered potential risks?

Tested locally so should be OK. Configs all updated to include new group.

@dblatcher - is it expected that we might create a newsletter without Series tags at all (Fronts?) if so, the logic in the email will need a tweak, as will the status updater.

## Images



![Screenshot 2023-09-25 at 14 57 24](https://github.com/guardian/newsletters-nx/assets/3277259/0e8b0ae5-24fe-47fb-87db-141a886c6533)
